### PR TITLE
Update introduction.rst

### PR DESCRIPTION
**Pull request by capitaine**
Added steps for installation of python modules 'magento' and 'pycountry' required by openERP 'magento_integration' module.

### CODE REVIEW

**Comment by sharoonthomas** [comment]
Thanks for the documentation. I think we will slightly refactor this and merge this as a single commit with you as the author.

File: doc/source/introduction.rst [hunk 1]
```diff
@@ -88,7 +88,17 @@ System >> Magento Connect >> Magento Connect Manager
...
 **Step 2: Installation of Magento Integration [OpenERP module]**
 
-**Downloading the module**
+**Preparation of the Python environment**
+
+1. If not already installed by the default python installation, download the pycountry module and install it in your python directory
+
+2. Download the `magento module <https://github.com/openlabs/magento>`_ and install it in your python directory (*)
+
+(*) it seems that in linux installations, Magento Integration takes care of 'magento module' and installs it when running the Magento Integration setup (see .5 below). So you have to install it yoursefl only if you are working under windows systems because for some reason Magento Integration lacks to install the 'magento module' (to be confirmed by developers)
```
Comment (priyankarani) [inline doc/source/introduction.rst:97]: more than 80 chars

typo s/yoursefl/yourself 

s/systems/system

and I think heading for this should be "Setup Python Environment"

File: doc/source/introduction.rst [hunk 2]
```diff
@@ -88,7 +88,17 @@ System >> Magento Connect >> Magento Connect Manager
...
+
+(if your openERP is installed on a Windows system, copy the above modules from YourPythonDir\\Lib\\site-packages to YourOpenErpDir\\Server\\server)
```
Comment (priyankarani) [inline doc/source/introduction.rst:99]: more than 80 chars

File: doc/source/introduction.rst (hunk 1 above)
Comment (shalabhaggarwal) [inline doc/source/introduction.rst:97]: IMO we are not setting up Python environment here. Instead this is actually "Setup of Python library dependencies"

As for windows, we have not tested the installation on windows so the behaviour cannot be confirmed. The best way suggested by community is appreciated.

**Comment by sharoonthomas** [comment]
Merged through #185

**Pull request closed without merging**